backend/routes/stocks.js

```diff
@@ -43,6 +45,22 @@ router.get("/stocks/:ticker", async (req, res) => {
                 ? (stock.dataValues.following = true)
                 : (stock.dataValues.following = false);
 
+            const liking = await Like.findOne({
+                where: { UserId, StockId }
+            });
+
+            liking
+                ? (stock.dataValues.liking = true)
+                : (stock.dataValues.liking = false);
```

Comment:
alternate way to write this: 
`stock.dataValues.liking = liking != null;`. Looks a bit cleaner to use boolean evaluation than writing separate true and false statements.

Reply:
Yeah, that looks way better.

Reply:
Can you think of a way without mutating the original stock object?